multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/FileSystemFileStorage.java

```diff
@@ -82,14 +82,9 @@ public void deleteFile(String id, String space) throws FileStorageException {
     }
 
     @Override
-    public void deleteFilesBySpace(String space) throws FileStorageException {
-        File spaceDirectory = getSpaceDirectory(space).toFile();
-        try {
-            if (spaceDirectory.exists()) {
-                FileUtils.deleteDirectory(spaceDirectory);
-            }
-        } catch (IOException e) {
-            throw new FileStorageException(MessageFormat.format(Messages.ERROR_DELETING_DIRECTORY, spaceDirectory), e);
+    public void deleteFilesBySpaces(List<String> spaces) throws FileStorageException {
+        for (String space : spaces) {
+            deleteFilesBySpace(space);
```

Comment:
If this method throws an exception, some files for spaces will not be deleted. Do you think it is worth wrapping it into a try-catch?